caesars-palace/blocks/slider-card/slider-card.css

```diff
@@ -0,0 +1,131 @@
+* {
+  box-sizing: border-box;
+}
+
+.slider-card {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  gap: 16px;
+  transform: translateX(0);
+  transition: transform 0.3s ease-out;
+}
+
+/* To prevent scrolling on the cards */
+.slider-card-wrapper {
```

Comment:
I think we want scrolling, or at least that animated effect on Desktop

Reply:
Done.